docassemble/ALAutomatedTestingTests/data/questions/al_testing_files_to_push.yml

```diff
@@ -70,7 +70,7 @@ content: |
     "scripts": {
       "test": "npm run langs_setup && npm run cucumber -- $@",
       "cucumber_base": "./node_modules/.bin/cucumber-js --require ./node_modules/docassemble-cucumber/lib/index.js",
-      "cucumber": "run_cucumber(){ npm run cucumber_base -- \"$@\" tests/features/*.feature; }; run_cucumber",
+      "cucumber": "run_cucumber(){ npm run cucumber_base -- \"$@\" tests/features/*.feature docassemble/*/data/sources/*.feature; }; run_cucumber",
```

Comment:
Hmm... do we still need `tests/features/*.feature` in here? You removed it in https://github.com/plocket/docassemble-cucumber/pull/213/commits/f70dd47a9cfe6c2864f4e7ced48c6ef1f0992869 but it's back here now.

Reply:
Thank you so much for catching this! No idea how this is here. Maybe I didn't push something.